python/cairo-ec/src/cairo_ec/circuits/ec_ops.cairo

```diff
@@ -19,3 +21,22 @@ func ec_double(x0: felt, y0: felt, a: felt) -> (felt, felt) {
 
     end:
 }
+
+// @dev Assert that a point is on the curve.
+func assert_on_curve(x: felt, y: felt, a: felt, b: felt) {
```

Comment:
nit: add ec in function name even for assert

Reply:
`ec_assert_on` ?